modules/ROOT/pages/guided-setup.adoc

```diff
@@ -25,12 +25,54 @@ This may be the US control plane, the EU control plane, the MuleSoft Government
 +
 If your Anypoint account has the organization administrator role, the guided setup tool can create the connected app for you.
 +
-Otherwise, create the connected app or ask an organization administrator to create it, then provide the app details so the guided setup can perform the authentication and authorization.
+Otherwise, create the connected app or ask an organization administrator to create it with these steps:
++
+.. Create a key pair in Salesforce.
++
+This key pair is created in Salesforce and used for communication between Salesforce and Anypoint Platform.
++
+... Click *Setup* > *Security* > *Certificate and Key Management* > *Create Self-Signed Certificate*.
+... Set *Label/Name* to `Anypoint`.
+... Disable *Exportable Private Key*.
+... Set *Key Size* to 2048 or larger.
+... *Save*.
+... Click *Download Certificate*.
++ 
+The public key is saved on your local machine and is required in the next step to create the connected app in Anypoint Platform.
++
+.. Create a connected app in Anypoint Platform.
+... Click *Access Management* > *Connected Apps* > *Create App*.
+... Choose a name.
+... Enable *App acts on behalf of a user*.
+... In the section *Grant types* enable *JWT Bearer*.
+... Copy the public key from the downloaded file and paste it into the public key text area.
++
+The certificate must have this format:
++
+----
+-----BEGIN CERTIFICATE-----
+PUBLIC KEY
+-----END CERTIFICATE-----
+----
+... Set *Website URL* to `+https://login.salesforce.com+`.
+... In the section *Redirect URIs* add `+http://localhost+`.
++
+Connected apps require redirect URIs and this configuration does not use them.
+... In the section *Who can use this application?* select *Members of this organization only*.
+... In the section *Scopes* add *Background Access* and *Full Access*.
+... *Save*.
++
+This shows a client ID and a client secret. This configuration uses the certificate and does not use the client secret. Save the client ID for the following steps.
 . Authenticate and authorize the connected app to access your Anypoint organization.
 +
 If your Anypoint account has the organization administrator role, provide your Anypoint Platform user and password, and the guided setup will create the app.
 +
-Otherwise, provide the app details.
+Otherwise, provide these app details:
++
+.. *Client ID*: Copy the client ID from the connected app you created.
+.. *IDP Issuer*: In your Anypoint Platform organization, click *Access Management* > *Identity Providers* > *SAML 2.0* > *Edit* and copy the *IDP Issuer* string.
+.. *Audience*: In your Anypoint Platform organization, click *Access Management* > *Identity Providers* > *SAML 2.0* > *Edit* and copy the *Audience* string.
```

Comment:
These are really too long to be just one step apiece, further complicating the step nesting issue if we break them out. 
Consider reorganizing this information for the next cycle.

Reply:
Shortened and rewrote for clarity.